README.md

```diff
@@ -61,6 +61,10 @@ $ stack build   # Compile Courseography and all Haskell dependencies (this will
 When running in production you should run `$ yarn build` instead of `$ yarn watch` to build the web assets.
 This will take longer but results in smaller asset files.
 
+### Running front-end tests
+
+1. Run 'yarn run test -- [insert the name of the test file]' to run front-end tests.
```

Comment:
I don't think you need "steps" here since there's only one step. A few sentences are okay.

However, I would split this up into `yarn run test` to run all tests, and then `yarn run test -- <path>` for running just a specific file or folder of tests.

Reply:
Hi David! Thank you so much for your feedback. I have incorporated it and would be happy to make further improvements if necessary.